extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -127,9 +130,41 @@ define('ext.wikia.adEngine.slotTweaker', [
 		}
 	}
 
+	function getRecoveredProviderContainer(providerContainer) {
+		var element = document.getElementById(win._sp_.getElementId(providerContainer.childNodes[0].id));
+		if (element && element.parentNode) {
+			return element.parentNode;
+		} else {
+			return null;
+		}
+	}
+
+	function tweakRecoveredSlot(adContainer) {
+		var className = 'tmpHeader';
+
+		if (browser.isIE() || browser.isEdge()) {
+			return;
```

Comment:
I'd investigate _why_ for IE and Edge we don't need these tweaks and maybe we can help somehow SP to recover the ad on Chrome and Firefox in the same way.